electron-app/magnifier/magnifier-main-rust.ts

```
@@ -0,0 +1,318 @@
+      const magnifierPath = join(
+        __dirname,
+        '../renderer/magnifier_window/index.html'
+      );
+      await this.magnifierWindow.loadFile(magnifierPath);
+    }
+
+    this.magnifierWindow.show();
+  }
+
+  private async startColorPicking(): Promise<string | null> {
+    return new Promise(async (resolve) => {
```

ESLint

[error] Promise executor functions should not be async.

(no-async-promise-executor)